interface/modules/custom_modules/oe-module-claimrev-connect/public/assets/js/skeleton-module.js

```diff
@@ -0,0 +1,39 @@
+(function(window) {
```

Comment:
Let's remove this file as it doesn't look like you are using it.

Reply:
done